src/parser/jobs/war/InnerRelease.js

```diff
@@ -100,19 +105,34 @@ export default class InnerRelease extends Module {
 			}))
 		}
 
-		if (this._missedGcds) {
+		if (this._missedGcds && !this._isRushing) {
```

Comment:
This is only checking the rush right during the `complete` event, and  skipping the entire suggestion if it is.

I'd probably move the rush check to the IR calculations itself, and skip counting missed GCDs if the rush is active.

Jokes just got up to that bit, you're already skipping the count if rushing. Can probably just remove this bit of the conditional then.